classes/StudentInsertData.cls

```diff
@@ -0,0 +1,41 @@
+public class StudentInsertData {
+      public StudentInsertData(){
+        
+        //Insert Student Record
+        Student__c s1 = new Student__c();
```

Comment:
Set record type and create individual Students with Teacher and Registrar record type.